client/src/Components/Navbar.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link, useNavigate, useParams } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faGlobe} from '@fortawesome/free-solid-svg-icons'
+import i18 from 'i18next'
+import { useAuth0 } from "@auth0/auth0-react";
+
+
+const Navbar = () => {
+  const [dropdownVisible, setDropdownVisible] = useState(false);
+  const {t} =  useTranslation();
+  const {lng} = useParams(); //capture the language from url
+  const navigate = useNavigate();
+  const { user, isLoading, isAuthenticated, loginWithRedirect, logout } = useAuth0();
+
+  const {ul1,ul2,ul3,ul4,ul5} = t('ulTranslation')
+  const {login,register,logOut} = t('logRegTranslation')
+  const {Technology,Music,Literature,Politics,Culture} = t('blogTopics')
+
+  useEffect(()=>{
+    const savedLang = localStorage.getItem('i18nextLng') || 'en';
+    i18.changeLanguage(savedLang);
+  },[])
+  
+  const handleLanguageChange = (lang) => {
+    const currentPath = window.location.pathname;
+    const newPath = currentPath.replace(`/${i18.language}`, `/${lang}`);
+    i18.changeLanguage(lang);
+    localStorage.setItem('i18nextLng', lang);
+    navigate(newPath);
+    setDropdownVisible(false);
+  };
+  
+  // const getLanguagePath = () => `/${i18.language}`
+
+  if(isLoading)
+  {
+    return <div>Loading...</div>
+  }
+
+  return (
+    <div className='gap-[1150px]'>
+
+      {/* This is the 1st navbar */}
+      <div className= "bg-green-700}">
+        <div className={`flex bg-green-700 gap-[1150px] ${i18.language === 'ur' ? 'gap-[1130px]' : 'gap-[1150px'}
```

Comment:
`gap-[1150px` isn't a matching `]` missing here?

Reply:
No it is matching, `by-default` it will be set on `1150px` but on selecting `ur` as the `lang` the gap will become of `1130px`

Reply:
please fix the syntax error, it also shows that testing is missing